docs/topics/secured-mission-intro.adoc

```diff
@@ -1,58 +1,28 @@
-for the alice user which enable alice to access the {WildFlySwarm}, {VertX} and {SpringBoot} based
-REST Booster deployments.
-
-==== The secured-swarm-endpoint Client
-The secured-swarm-endpoint is a bearer-only type of client that requires a swarm-admin role for accessing the
-associated resources that represents the secured {WildFlySwarm} Booster.
+app:: 
+The `app` module exposes the REST Service using {runtime-name}.
 
-==== The secured-vertx-endpoint Client
-The secured-vertx-endpoint is a bearer-only type of client that requires a vertx-admin role for accessing the
-associated resources that represents the secured {VertX} Booster.
+sso::
+The `sso` module is a submodule link to the link:https://github.com/obsidian-toaster-quickstarts/redhat-sso[redhat-sso] project that contains the OpenShift objects required to deploy the {RHSSO} Server 7.0 as well as a Java command line client driver to test access to the secured endpoint.
 
-==== The secured-springboot-endpoint Client
-The secured-springboot-endpoint is a bearer-only type of client that requires a springboot-admin role for accessing the
-associated resources that represents the secured {SpringBoot} Booster.
+= {RHSSO} Realm Model
 
-== The Users
-The realm model for the secured Boosters includes an admin and alice user.
+The `master` realm is used to secure this booster. It is defined using the `src/resources/master.json` JSON description. There are four pre-configured application client definitions that provide a model for command line clients and the three secured REST endpoints. There are also two pre-configured users in the {RHSSO} `master` realm that can be used to validate various authentication and authorization outcomes:  `admin` and `alice`.
 
-=== The admin User
-The admin user has a password of admin, and is the realm adminstrator. This user has full access to the RH SSO administration
-console, but none of the role mappings that are required to access the secured endpoints. You can use this user to
-illustrate the behavior of an authenticated, but unauthorized user.
+== {RHSSO} Users
+The realm model for the secured boosters includes two users:
 
-=== The alice User
-The alice user is the canonical application user that will demonstrate successful authenticated and authorized access
- to the secured endpoints. A representation of the role mappings is provided in this decoded JWT bearer token
+admin::
+The `admin` user has a password of `admin`, and is the realm administrator. This user has full access to the {RHSSO} administration console, but none of the role mappings that are required to access the secured endpoints. You can use this user to illustrate the behavior of an authenticated, but unauthorized user.
 
-[source,json]
+alice:: 
+The `alice` user is the canonical application user that will demonstrate successful authenticated and authorized access to the secured endpoints. A representation of the role mappings is provided in this decoded JWT bearer token
```

Comment:
For the `admin` user, we tell what the password is; maybe we should also tell that for the `alice` user. FYI, it's literally `password`.